src/test/java/baseballGame/BaseballGameTest.java

```diff
@@ -0,0 +1,42 @@
+package baseballGame;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class BaseballGameTest {
+
+    @Test
+    void createBallsTest() {
+
+        Balls balls = new Balls();
+        ArrayList<Integer> computerBalls = balls.createBalls();
+
+        assertThat(computerBalls.size()).isEqualTo(3);
+    }
+
+    @Test
+    void compareBallsTest() {
+
+        ArrayList<Integer> computerBalls = new ArrayList<>();
+        ArrayList<Integer> myBalls = new ArrayList<>();
+        computerBalls.add(8);
+        computerBalls.add(7);
+        computerBalls.add(5);
+        myBalls.add(5);
+        myBalls.add(7);
+        myBalls.add(8);
```

Comment:
```suggestion
        List<Integer> computerBalls = List.of(8,7,5);
        List<Integer> computerBalls = List.of(5,7,8);
```

java 11 버전에서 지원하는 API를 적극 사용해보면서 익혀보는건 어떨까요?
java 9 버전부터 List.of 라는 메소드를 지원하기 시작했고, List.of롤 통해 List를 초기화하기 쉬워졌습니다. 🙂